File: src/main/webapp/js/app/master/pengirim.service.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
(function() {

	var services = angular.module('je-master-pengirim-service', [ 'ngResource' ]);

	
	services.factory('PengirimListFactory', [ '$resource', function($resource) {

		return $resource('service/master/pengirim', {}, {
			query : {
				method : 'GET',
				isArray : false
			},
			create : {
				method : 'POST'
			}
		});

	} ]);
	
	services.factory('PengirimEditFactory', [ '$resource', function($resource) {

		return $resource('service/master/pengirim/:nip', {}, {
			show : {
				method : 'GET',
				params: {nip: '@nip'}
			},
			update : {
				method : 'PUT',
				params :{nip: '@nip'}
			},
			remove : {
				method : 'DELETE',
				params :{nip: '@nip'}
			}
		});

	} ]);
	
})();

File: src/main/java/com/je/spring/rest/master/service/PengirimService.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.je.spring.rest.master.service;

import com.je.spring.rest.master.model.Pengirim;
import java.util.List;
/**
 *
 * @author User
 */
public interface PengirimService {
    public List<Pengirim> getAll();
    public Pengirim getById(int id);
    public long count();
    public void insert(Pengirim pengirim);
    public void update(Pengirim pengirim);
    public void delete(Pengirim pengirim);
}
